LayoutTests/fast/events/contextmenu-items-match-node-receiving-contextmenu-event.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <style>
+        body {
+            margin: 0;
+            padding: 0;
+        }
+        video {
+            width: 400px;
+            height: 300px;
+            position: absolute;
+            left: 100px;
+            top: 100px;
+        }
+        #overlay {
+            width: 50px;
+            height: 50px;
+            background: red;
+            position: absolute;
+            z-index: 1;
+        }
+    </style>
+</head>
+<body>
+    <p>The node which receives a 'contextmenu' event should be the node hit-tested when determining which items to show.</p>
+    <video id="video" src="resources/test.mp4"></video>
```

Comment:
nit, mainly because 188 KB is not tiny, I feel like you could get away with any of the other pre-existing `.mp4` test resources? Instead of checking in a new file?

```suggestion
    <video id="video" src="../../media/resources/white.mp4"></video>
```

Reply:
I know this probably is not a super likely scenario, but I decided against this originally out of concern that, if the media tests are ever updated to not use that video and it ends up being removed, it won't be obvious that this test was relying on it as well because the test would continue to pass.

Reply:
I think I'll just add some JS to fail the test if the file can't be loaded, just to be safe. That way we can reuse an existing file without any false passes.